cmd/kops/update_cluster.go

```diff
@@ -172,6 +175,39 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		glog.Infof("Using SSH public key: %v\n", c.SSHPublicKey)
 	}
 
+	// TODO: investegate if better to use kops create cecret to set ca cert and private key: v2 implementation when working
```

Comment:
Thanks! If you need any help:

1. there is developer office hours Friday.  See project README
2. ping one of the contributors on slack. 

Again I appreciate that you are open to the feedback, and are willing to help!